cheatsheets/Key_Management_Cheat_Sheet.md

```diff
@@ -108,11 +108,11 @@ Review `NIST SP 800-57` (Recommendation for Key Management) for recommended guid
 
 ### Memory Management Considerations
 
-Keys stored in memory for a long time can become "burned in". This can be mitigated by splitting the key into components that are frequently updated. `NIST SP 800.57`).
+Keys stored in memory for a long time can become "burned in". This can be mitigated by splitting the key into components that are frequently updated. `NIST SP 800-57`).
```

Comment:
If would be useful to include the relevant SP 800-57 section numbers as well. It might even have the side effect of having some people read the special publication, or at least the relevant parts of it.  Ditto with the other references to this as well.